tasks/main.yml

```diff
@@ -42,12 +42,18 @@
     - cartridge-replicasets
     - cartridge-config
 
+- name: Get one instance from each physical machine
+  set_fact:
+    machine_hosts_list: '{{ hostvars | get_one_not_expelled_instance_for_machine(play_hosts) }}'
```

Comment:
Maybe `machine_hostnames`?

Reply:
But it's not a hostname, it's names of Ansible hosts

Reply:
So, it's smth like `instances_to_install_package`?

Reply:
`single_instances_for_each_machine`